Mathlib/Analysis/Complex/Hadamard.lean

```diff
@@ -421,5 +426,229 @@ lemma norm_le_interp_of_mem_verticalClosedStrip' (f : ℂ → E) {z : ℂ} {a b
         simp only [mem_preimage, one_re, mem_singleton_iff, comp_apply,
           and_self]
 
+
+/--An auxiliary function to prove the general statement of Hadamard's three lines theorem.-/
+def scale (f: ℂ → E) (l u : ℝ) : ℂ → E := fun z ↦ f (l + z • (u-l))
+
+/--The transformation on ℂ that is used for `scale` maps the strip ``re ⁻¹' (l,u)``
+  to the strip ``re ⁻¹' (0,1)``-/
+lemma scale_mapsto_dom {l u : ℝ} (hul: l<u) {z: ℂ} (hz: z ∈ verticalStrip 0 1) :
+    l + z * (u-l)  ∈ verticalStrip l u := by
```

Comment:
I think this should either be named something like `scale_mem_verticalStrip_of_mem_verticalStrip`, or else actually use `Set.MapsTo`.

Reply:
I've changed the name (and the following ones) as suggested.
While doing so, I remembered that technically this lemma refers to the map `\varphi: \mathbb{C} \to \mathbb{C}` given by `z \mapsto l + z * (u - l)`, whereas scale maps a function f to a function `(scale f) z = f (\varphi z)`. I am not sure whether this is fine or I should define this `\varphi` separately (possibly with a nicer name) and make the lemma about it.

Reply:
Yeah I think that's fine.